Bergado_Shamma/index.html

```diff
@@ -0,0 +1,230 @@
+          </p>
+        </li>
+        <li class="singing">
+          <a href="https://www.karafun.com/karaoke/styles/popmusic/">
+            Singing
+          </a>
+          <p id="p">
+            "Songs are like tattoos—you know, they're with you forever." 
+            - Bob Dylan
+          </p>
+        </li>
+      </ul>
+    </div>
+
+    <div class="works">
+      <h2>My works</h2>
+      <ol>
+        <li class="psych">
+          <a href="./psychologist.html">Psychologist</a>
+        </li>
+        <li class="teach">
+          <a href="./teacher.html">Teacher</a>
+        </li>
+        <li class="it">
+          <a href="./it.html">IT</a>
+        </li>
+        <li class="nurse">
+          <a href="./nurse.html">Nurse</a>
+        </li>
+        <li class="law">
+          <a href="./lawyer.html">Lawyer</a>
+        </li>
+      </ol>
+    </div>
+
+    <div class="my-goals">
+      <h2>My Goals</h2>
+      <ol>
+        <li>To be a Developer</li>
+        <li>To be mentally and financially stable</li>
+        <li>To give my family a decent house to live</li>
+        <li>To travel inside and outside this country</li>
+        <li>To have a family, be a good mother and wife.</li>
+      </ol>
+      <h3>Add a comment here</h3>
+      <form class="add-comment">
+        <input
+          type="text"
+          name="name"
+          id="name"
+          placeholder="Enter your name"/>
+        <textarea
+          name="textarea-for-comment"
+          id="textarea_for_comment"
+          placeholder="Add your comment here">
+        </textarea>
+        <button type="button" name="comment" id="comment">Comment</button>
+        <button type="button" id="sort_asc" onclick="sortComments('asc')">
+          Sort by Date (Ascending)
+        </button>
+        <button type="button" id="sort_desc" onclick="sortComments('desc')">
+          Sort by Date (Descending)
+        </button>
+      </form>
+
+      <div class="comments-of-team">
+        <h3>Comments</h3>
+      </div>
+
+      <div class="comments-of-team2">
+        <h3>Hardcoded Comments</h3>
+        <p>Name: Paul Angelo Reyes</p>
+        <p>Comment: You can do it!</p>
+        <p>Name: John Ruiz Parayno Reyes</p>
+        <p>
+          Comment:Every step you take towards your goals brings you closer to
+          your dreams; keep pushing forward, and soon your hard work will pay
+          off.
+        </p>
+        <p>Name: Mary Nicole Perillo Norona</p>
+        <p>
+          Comment: Your goals should be your motivation and inspiration in 
+          life.
+          Just keep going and keep pushing yourself in achieving your goals.
+          Soon, all your hardwork will be worth it.
+        </p>
+        <p>Name: Daisy Cangayda Morcoso</p>
+        <p>
+          Comment: Your goals perfectly represent your values and desires. 
+          I'm
+          thrilled to see how you bring them to life!
+        </p>
+      </div>
+    </div>
```

Comment:
This change is not required, but it would be visually better to have each comment's elements (name, comment, and date) in one container. Then each comment should be in a separate container, and then styled for better organization and to avoid a cluttered appearance.